frontend/app/src/store/defi/getters.ts

```diff
@@ -1398,16 +1398,12 @@ export const getters: Getters<DefiState, DefiGetters, RotkehlchenState, any> = {
     },
 
   yearnVaultsAssets:
-    (
-      { yearnVaultsBalances, yearnVaultsV2Balances },
-      _dg,
-      _rs,
-      { 'balances/assetSymbol': assetSymbol }
-    ) =>
+    ({ yearnVaultsBalances, yearnVaultsV2Balances }, _dg, _rs) =>
```

Comment:
since we removed the root getters, it would be safe to now remove the unused `_dg`/`_rs` parameters right?